src/integrationTest/java/org/opensearch/test/framework/matcher/SearchHitContainsFieldWithValueMatcher.java

```diff
@@ -49,7 +49,7 @@
 			mismatchDescription.appendText("Source document is null, is fetch source option set to true?");
 			return false;
 		}
-		if(!source.containsKey(fieldName)) {
+		if(source.containsKey(fieldName) == false) {
```

Comment:
nit: This change should not be needed

Reply:
Please correct me if I am wrong, but I thought that some documents define the convention to use expressions like `source.containsKey(fieldName) == false` instead of `!source.containsKey(fieldName)`

Reply:
interesting...I'm not aware about such documents.. If you have one handy could you please link it here

Reply:
> There is an implicit rule that negative boolean expressions should use the form foo == false instead of !foo for better readability of the code. While this isn't strictly enforced, if might get called out in PR reviews as something to change.

Please see https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md . 

But this file is placed in the main OpenSearch repository and not in the security plugin repository. Therefore I am not sure if the rule is applicable.